reactive/kubernetes_control_plane.py

```diff
@@ -3366,8 +3356,6 @@ def get_dns_provider():
         if not dns_provider:
             if "core-dns" in valid_dns_providers:
                 dns_provider = "core-dns"
```

Comment:
small knit; this `if` isn't needed anymore.
```suggestion
            dns_provider = "core-dns"
```